packages/kbn-plugin-helpers/package.json

```diff
@@ -10,12 +10,15 @@
   "author": "Spencer Alger <email@spalger.com>",
   "license": "Apache-2.0",
   "dependencies": {
+    "@babel/core": "^7.4.4",
+    "@kbn/babel-preset": "1.0.0",
```

Comment:
__note:__ with this change on the latest master I'm getting `error Couldn't find package "@kbn/babel-preset@1.0.0" required by "@kbn/plugin-helpers@link:../../kibana/packages/kbn-plugin-helpers" on the "npm" registry.` on `yarn kbn bootstrap` (right after `yarn kbn clean`).  Am I doing something wrong? :)

Reply:
That package is still at `1.0.0` locally, and it works for me, but we've been doing a fair amount of upgrades lately and I've had to wipe out my node_modules directory more frequently than desired because `yarn` isn't able to properly patch the directory correctly.

Reply:
That's weird, `yarn kbn clean` should remove `node_modules`, but that still doesn't help. Anyway, if it becomes annoying, I'll try to investigate and give you folks more details :)